test/support/api/outreach.js

```diff
@@ -1,18 +1,27 @@
 import _ from 'underscore';
-import { getResource, getRelationship } from 'helpers/json-api';
+import { v4 as uuid } from 'uuid';
+import { getResource, getRelationship, mergeJsonApi } from 'helpers/json-api';
+
 import { getPatient } from './patients';
 
 const TYPE = 'outreach';
 
-const fxOutreach = {
-  id: '11111',
-  phone_end: '1234',
-};
+const fxOutreach = { phone_end: '1234' };
+
+export function getOutreachStatus(data) {
+  const defaultRelationships = {
+    'patient': getRelationship(getPatient()),
+  };
+
+  const resource = getResource(fxOutreach, TYPE, defaultRelationships);
+
+  data = _.extend({ id: uuid() }, data);
+
+  return mergeJsonApi(resource, data, { VALID: { relationships: _.keys(defaultRelationships) } });
+}
```

Comment:
`getOutreachStatus()` is a copy of `getAction()` with some modifications to fit the outreach status data.